evennia/settings_default.py

```diff
@@ -201,6 +201,14 @@
 MAX_COMMAND_RATE = 80
 # The warning to echo back to users if they send commands too fast
 COMMAND_RATE_WARNING ="You entered commands too fast. Wait a moment and try again."
+# Determine how large of a string can be sent to the server in number
+# of characters. If they attempt to enter a string over this character
+# limit, we stop them and send a message. Set to None by default. To
+# change it, just set it to a number of characters - ie, 6000 to be
+# roughly two pages of text.
+MAX_CHAR_LIMIT = None
```

Comment:
I think it may be a reasonable thing to actually set a sane default max here - it's good for people that don't bother with the settings. People can always up the limit manually later if they think it's limiting. I don't know what the max would be though - it would need to be a value you rarely hit yet not so large you run into slowdowns. 6000 sounds like a lot to me, but maybe it's not ...?